index.html

```diff
@@ -10,97 +10,133 @@
 
 </head>
 <body>
-
+      
        <header class="header-bar">
 
         <nav class="nav-bar"><h3 class="Left-Logo">My Logo</h3>
```

Comment:
[OPTIONAL] Kindly wrap the logo in `a` tag with `href=#` so that a User can go back to the top of page anytime by clicking the logo 😃